### PR TITLE
Feat/rating

### PR DESCRIPTION
**Pull request by tonymike99**
I've added the rating component.

### CODE REVIEW

**Comment by DivyanshSareen** [comment]
The code looks good. The commits are done nicely. Good to merge

**Review by DivyanshSareen — APPROVED** [review]
Code looks good. The commits are done nicely. Good to merge

File: docs.html [hunk 1]
```diff
@@ -1912,6 +1911,35 @@ <h3 class="component-subtitle">Examples of Headings</h3>
...
+                        style="
+                            width: 495px;
+                            height: 372px;
+                            border: 0;
```
Comment (parulgupta2001) [inline docs.html:1920]: border : 0  does not exist.  It should be border : none

**Review by parulgupta2001 — APPROVED** [review]
make the changes as suggested below. Otherwise code looks ok

**Review by zoyatanweer — APPROVED** [review]
- Very clean and easily understandable code. 
- Great usage of comments and nice naming convention of classes
- Good to merge!

File: docs.html (hunk 1 above)
Comment (tonymike99) [reply docs.html:1920]: It was a part of the carbon code.
And border:0 is same as border:none :)

**Pull request merged**